go/hyperkit.go

```diff
@@ -248,6 +248,7 @@ func (h *HyperKit) Start(cmdline string) error {
 
 func (h *HyperKit) execute(cmdline string) error {
 	var err error
+	h.log.Printf("hyperkit: execute %#v", h)
```

Comment:
not sure how to comment on commit messages, but it should read something like "go: Improve logging" or some such. I read "add logs" and didn't know what it meant.